pos_journal_pin/doc/index.rst

```diff
@@ -13,26 +13,26 @@ Configuration
 Manager's PIN
 -------------
 
-Set a PIN for a POS manager as it's explained in `Confirm POS action by PIN` module
-As Administrator do next:
+Set a PIN for a POS manager as it's explained in ``Confirm POS action by PIN`` module.
+As Administrator please follow next scenario:
 
 Journal
 -------
 
   * Open ``[[Point of Sale]] >> Payment Methods`` menu
   * Choose a journal or create one, be sure the journal is active in POS
-  * Activate **Manager's Permission** field
+  * Activate **POS Manager's Permission** field
```

Comment:
Please return it back. Here **Manager's Permission** is the field name so if we change it here then we have to change it in the code too